data/libs/autoload.lua

```diff
@@ -98,6 +98,27 @@ table.merge = function(a, b, transformer)
 	return a
 end
 
+-- Copy table recursively using pairs()
+--
+-- Does not copy metatable
+---@generic T
+---@param t T
+---@return T
+table.deepCopy = function(t)
```

Comment:
For consistency with the Lua standard library conventions, this should be `table.deepcopy`.